openapi/go-client-generated/docs/BlocksApi.md

```diff
@@ -82,3 +83,40 @@ No authorization required
 
 [[Back to top]](#) [[Back to API list]](../README.md#documentation-for-api-endpoints) [[Back to Model list]](../README.md#documentation-for-models) [[Back to README]](../README.md)
 
+# **BlocksIdPayloadGet**
+> BlockPayload BlocksIdPayloadGet(ctx, id, optional)
+Get Block Payload by Block ID.
+
+Get a block payload data by the provided block ID.
+
+### Required Parameters
+
+Name | Type | Description  | Notes
+------------- | ------------- | ------------- | -------------
+ **ctx** | **context.Context** | context for authentication, logging, cancellation, deadlines, tracing, etc.
+  **id** | [**string**](.md)| A block ID. | 
+ **optional** | ***BlocksApiBlocksIdPayloadGetOpts** | optional parameters | nil if no parameters
+
+### Optional Parameters
+Optional parameters are passed through a pointer to a BlocksApiBlocksIdPayloadGetOpts struct
+Name | Type | Description  | Notes
+------------- | ------------- | ------------- | -------------
+
+ **expand** | [**optional.Interface of []string**](string.md)| A comma-separated list indicating which properties of the content to expand. | 
+ **select_** | [**optional.Interface of []string**](string.md)| A comma-separated list indicating which properties of the content to return. | 
```

Comment:
Nit: for some reason, Github can't format it properly. 
![Uploading image.png…]()